ntriples/shared/src/main/scala/org/w3/banana/io/NTriplesWriter.scala

```diff
@@ -33,17 +33,17 @@ class NTriplesWriter[Rdf <: RDF](implicit val ops:RDFOps[Rdf]) extends RDFWriter
     }
   )
 
-  def write(graph: Rdf#Graph, os: OutputStream, base: String): Try[Unit] = Try {
+  def write(graph: Rdf#Graph, os: OutputStream, base: String, prefixes: Prefix[Rdf]*): Try[Unit] = Try {
```

Comment:
Given that in NTriples it makes no sense to set prefixes, it would be good to set a default value here (if that's ok with Scala)

Reply:
I'm not sure i completely understand what you mean here. There is an overloaded function on the RDFWriter type, taking no Prefixes and forwards to this implementation. In the case of NTriples, the prefixes would be ignored, but for a symmetry the function signature is present.